### ISSUE TITLE
Params not sent when joining a Channel

### ISSUE THREAD

**Issue by optikfluffel** [davidstump/SwiftPhoenixClient#87]
It looks to me like #85 forgot about [`/// The params sent when joining the channel`](https://github.com/davidstump/SwiftPhoenixClient/blob/master/Sources/Channel.swift#L32-L33).

When I try to:

```Swift
let channel = socket.channel("room:secret", params: ["token": "Super Secret Room Token"])

channel.join()
    .receive("ok") { payload in print("Channel Joined", payload) }
    .receive("error") { payload in print("Failed to join", payload) }
    .receive("timeout") { payload in print("Networking issue...", payload) }
```

I never seem to receive the `token` server side.

**Comment by danielrees18** [comment]
Good catch! I'll work on getting a patch fix out shortly.

**Comment by danielrees18** [comment]
Available now in release [0.8.1](https://github.com/davidstump/SwiftPhoenixClient/releases/tag/0.8.1)

Let me know if you continue to have any problems and thanks for helping with this project!
